### ISSUE TITLE
Unable to retreive JOB by jobname using RSE profile

### ISSUE THREAD

**Issue by aymeric56** [IBM/zopeneditor-about#452]
## Development environment used

- Z Open Editor version: 4.5.1
- Editor Platform
  - [X] Visual Studio Code
  - [ ] Red Hat CodeReady Workspaces
  - [ ] Eclipse Che
  - [ ] Standalone Theia
- Editor Platform Version: Visual Studio Code 1.94.2
- Operating System: macOS Sequoia 
- Java Version : >java -version
openjdk version "21.0.4" 2024-07-16 LTS
IBM Semeru Runtime Open Edition 21.0.4.0 (build 21.0.4+7-LTS)
Eclipse OpenJ9 VM 21.0.4.0 (build openj9-0.46.0, JRE 21 Mac OS X aarch64-64-Bit 20240716_229 (JIT enabled, AOT enabled)
OpenJ9   - 1a6f6128aa
OMR      - 840a9adba
JCL      - 7d844187b25 based on jdk-21.0.4+7)
- [X] Related to RSE API?
  - RSE API Plugin version: ?
  - Zowe CLI version: ?
  - Node.js version: ?
  - Zowe Explorer: 2.18.0
- Logs attached yes
[Zowe Explorer.log](https://github.com/user-attachments/files/17340719/Zowe.Explorer.log)
[Z Open Editor.log](https://github.com/user-attachments/files/17340725/Z.Open.Editor.log)
## Problem Description

Error message:
Retrieving response from zowe.GetJobs Error: Rest API failure with HTTP(S) status 500

Detailed steps for reproducing the problem:

1. Submit a job
2. Click on the popup at the bottom right corner specifying the jobname
3. get the error saying it can't retreive the job

### Observed behavior

- Retrieving response from zowe.GetJobs Error: Rest API failure with HTTP(S) status 500

### Expected behavior

- List the job on the JOBS tab. If I select another filter like all the jobs I'm the owner, with Prefix:* and Status:*, it's working as expected: I get all my jobs.

**Comment by aymeric56** [comment]
I did have a look to the swagger documentation of the RSEAPI server:
the request in error made by the RSE plugin is this one:
/api/v1/jobs?owner=*&jobid=JOB06683
but I can't see in the swagger document it's possible to do it. On this path, it's only possible to put the jobId in a header named Included-Attributes. But I've tried it, with no result.
I've tried this pass, which is working:
/api/v1/jobs/%2A/JOB06683
